codespell_lib/data/dictionary.txt

```diff
@@ -7208,7 +7210,7 @@ counterfit->counterfeit
 counterintuive->counter intuitive
 counterpar->counterpart
 counterpoart->counterpart
-countinueq->continueq
+countinueq->continue
```

Comment:
If it's used in Linux headers, perhaps suggest both versions, probably Linux first.

Reply:
added